pkg/plugins/platform/local_test.go

```diff
@@ -131,6 +131,24 @@ func TestLocalStart(t *testing.T) {
 
 		cleanup()
 	}
+
+	// try to start a site that is already running
+	err = app.Init(TestSites[0].Dir)
+	assert.NoError(err)
+
+	err = app.Start()
+	assert.EqualError(err, fmt.Sprintf("a site with the name %s is already running", TestSites[0].Name))
```

Comment:
This check was intended to prevent a site that is already running from being started again, but I hadn't considered the use case for re-running start on an existing site. It is now removed.